notebooks/fairness-dashboard-loan-allocation.ipynb

```diff
@@ -302,24 +309,18 @@
   }
  ],
  "metadata": {
+  "interpreter": {
```

Comment:
Is this your local env setup?
Probably not good to check in.

Reply:
Arghh, hadn't spotted that something 'helpfully' did that. Thanks. Will remove.